Styles/utility.css

```diff
@@ -62,6 +62,7 @@ li{
     align-items: center;
     width: 100%;
     flex-wrap: wrap;
+    gap: 4%;

```

Comment:
suggestion , you can use `rem` here

Reply:
Thanks!  will consider this utility class changes in next  PR.